app/assets/javascripts/components/gtm-form-listener.js

```diff
@@ -0,0 +1,35 @@
+function GTMFormListener ($form, $dataLayer) {
+  this.$form = $form
+  this.dataLayer = $dataLayer || window.dataLayer
+}
+
+GTMFormListener.prototype.handleSubmit = function (event) {
+  var eventName = this.$form.dataset.gtm
+  var message = {}
+
+  var inputElements = this.$form.querySelectorAll('input:checked')
+  inputElements.forEach(function (element) {
+    if (element.value.length > 0) {
+      message = {}
+      message[eventName] = {}
+      message[eventName][element.name] = element.value
+      this.dataLayer.push(message)
+    }
+  }, this)
+}
+
+GTMFormListener.prototype.init = function () {
+  var $form = this.$form
```

Comment:
I think we may as well just use this.$form in the two places this is used in this method?

Reply:
Fixed in 80e2f7107c683c971a0a6273d97f089659ed63c1